aleph-client/src/lib.rs

```diff
@@ -122,6 +124,27 @@ pub fn send_xt(connection: &Connection, xt: String, xt_name: &'static str, tx_st
     );
 }
 
+pub fn try_send_xt(
+    connection: &Connection,
+    xt: String,
+    xt_name: &'static str,
+    tx_status: XtStatus,
+) -> ApiResult<()> {
+    let block_hash = connection
+        .send_extrinsic(xt, tx_status)?
```

Comment:
This blocks until the transaction's status becomes `tx_status`, correct? Is there any timeout on that?

Reply:
I want to add async api here soon